p4sdk/README.md

```diff
@@ -66,4 +66,11 @@ make image
 ```
 
 
+### To run a container with the P4 artifacts
 
+
+```
+podman run -d --privileged -v /lib/modules/5.14.0-425.el9.aarch64:/lib/modules/5.14.0-425.el9.aarch64 localhost/intel-ipu-p4-sdk:latest
```

Comment:
could we create a pod spec in replace of this?

Reply:
Planning to do this in a separate PR to limit the scope. Here we are only looking at running infrap4d inside a container.